test/web3/schema.json

```diff
@@ -0,0 +1,163 @@
+    "params3": [
+      "0xbb3a336e3f823ec18197f1e13ee875700f08f03e2cab75f0d0b118dabb44cba0"
+    ],
+    "params4": ["params", "0x0"],
+    "params5": ["0x29c", "0x0"],
+    "params6": ["params"],
+    "params7": "0x5BAD55",
+    "params8": "0xd46e8dd67c5d32be8d46e8dd67c5d32be8058bb8eb970870f072445675058bb8eb970870f072445675",
+    "params9": "0xc94770007dda54cF92009BFF0dE90c06F603a09f",
+    "submitworkparams": [
+      "0x0000000000000001",
+      "0x1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef",
+      "0xD1FE5700000000000000000000000000D1FE5700000000000000000000000000"
+    ],
+    "getstorageatparams": [
+      "0x295a70b2de5e3953354a6a8344e616ed314d7251",
+      "0x6661e9d6d8b923d5bbaab1b96e1dd51ff6ea2a93520fdc9eb75d059238b8c5e9",
+      "0x65a8db"
+    ],
+    "getcodeparams": ["0x06012c8cf97bead5deae237070f9587f8e7a266d", "0x65a8db"],
+    "EstimateTransaction": {
+      "from": "0xb60e8dd61c5d32be8058bb8eb970870f07233155",
+      "to": "0xd46e8dd67c5d32be8058bb8eb970870f07244567",
+      "gas": "0x76c0",
+      "gasPrice": "0x9184e72a000",
+      "value": "0x9184e72a",
+      "data": "params8"
+    },
+    "FilterGetLogs": {
+      "blockHash": "0x7c5a35e9cb3e8ae0e221ab470abae9d446c3a5626ce6689fc777dcffcab52c70",
+      "topics": [
+        "0x241ea03ca20251805084d27d4440371c34a0b85ff108f6bb5611248f73818b80"
+      ]
+    },
+    "FilterNewFilter": {
+      "topics": [
+        "0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef"
+      ]
+    },
+    "Block": {
+      "__required": [],
+      "number": "Q",
+      "hash": "D32",
+      "parentHash": "D32",
+      "nonce": "D",
+      "sha3Uncles": "D",
+      "logsBloom": "D",
+      "transactionsRoot": "D",
+      "stateRoot": "D",
+      "receiptsRoot": "D",
+      "miner": "D",
+      "difficulty": "Q",
+      "totalDifficulty": "Q",
+      "extraData": "D",
+      "size": "Q",
+      "gasLimit": "Q",
+      "gasUsed": "Q",
+      "timestamp": "Q",
+      "transactions": ["DATA|Transaction"],
+      "uncles": ["D"]
+    },
+    "Transaction": {
+      "__required": [],
+      "hash": "D32",
+      "nonce": "Q",
+      "blockHash": "D32",
+      "blockNumber": "Q",
+      "transactionIndex": "Q",
+      "from": "D20",
+      "to": "D20",
+      "value": "Q",
+      "gasPrice": "Q",
+      "gas": "Q",
+      "input": "D"
+    },
+    "Receipt": {
+      "__required": [],
+      "transactionHash": "D32",
+      "transactionIndex": "Q",
+      "blockHash": "D32",
+      "blockNumber": "Q",
+      "cumulativeGasUsed": "Q",
+      "gasUsed": "Q",
+      "contractAddress": "D20",
+      "logs": ["FilterChange"]
+    },
+  
+    "FilterChange": {
+      "__required": [],
+      "removed": "B",
+      "logIndex": "Q",
+      "transactionIndex": "Q",
+      "transactionHash": "D32",
+      "blockHash": "D32",
+      "blockNumber": "Q",
+      "address": "D20",
+      "data": "Array|DATA",
+      "topics": ["D"]
+    },
+    "methods1": {
```

Comment:
Like my comment above on params, it would be good to give these methods objects descriptive names.